lib/Conversion/TorchToLinalg/Random.cpp

```diff
@@ -95,14 +120,6 @@ class ConvertAtenUniformOp : public OpConversionPattern<AtenUniformOp> {
         loc, rewriter.getI64IntegerAttr(6364136223846793005));
     Value incrementStep = rewriter.create<arith::ConstantOp>(
         loc, rewriter.getI64IntegerAttr(1442695040888963407));
```

Comment:
I would move 

https://github.com/llvm/torch-mlir/blob/10cf74cd381fe91c057fb71b75308d974b9d13e1/lib/Conversion/TorchToLinalg/Random.cpp#L111-L122

to `getNextTemp`, since it's all related. This also avoid repeating the multiplier and increment values multiple times in this file